pxc-80-backup/Dockerfile

```diff
@@ -0,0 +1,87 @@
+FROM registry.access.redhat.com/ubi7/ubi-minimal
```

Comment:
@delgod  Should we use 'Dockerfile.k8s' as a name of file if we used ubi-minimal?

Reply:
up to you, we need only ubi image, so it has no sense to create centos image with separate Dockerfile